releasenotes/notes/qiskit-algorithms-incorporated-421554a4ff547d0d.yaml

```diff
@@ -0,0 +1,59 @@
+---
+prelude: >
+    This release includes the migration of a subset of Qiskit Algorithms features to Qiskit Machine Learning.
+    This ensures continued enhancement of essential features for Qiskit Machine Learning following the end
+    of official support for Qiskit Algorithms. Therefore, Qiskit Machine Learning will no longer depend on
+    Qiskit Algorithms, possibly introducing breaking changes in import structures. Some quick-fixes are
+    described below.
+
+features:
+  - |
+    Migrated essential Qiskit Algorithms features to Qiskit Machine Learning:
+    - `qiskit_algorithms/gradients` -> `qiskit_machine_learning/gradients`
+    - `qiskit_algorithms/optimizers` -> `qiskit_machine_learning/optimizers`
+    - `qiskit_algorithms/state_fidelities` -> `qiskit_machine_learning/state_fidelities`
+    - Partial merge of `qiskit_algorithms/utils` with `qiskit_machine_learning/utils`
+  - |
```

Comment:
Do you want to noted that this was a subset of the gradients (reverse and finite-diff for instance were not copied over)

I do not think mention of unit tests is necessary for normal end-users.